step.yml

```diff
@@ -0,0 +1,297 @@
+        Examples: 
+        For class filtering strategy:
+        
+        `class com.android.foo.ClassA`, `class com.android.foo.ClassB`,`class com.android.foo.ClassC`
+        
+        For package filtering strategy:
+        
+        `package com.android.foo`
+        
+        For annotation filtering strategy:
+        
+        `annotation com.android.foo.MyAnnotation`
+        
+        For size filtering strategy:
+        
+        `size small`,`size medium`,`size large`
+      description: |
+        Provide key-value pairs of filters to run tests: class, package, annotation, size
+        
+        For example: class com.foo.AddToCartClass,class com.foo.CheckoutClass
+        
+        package com.foo
+        
+        Learn more.(https://www.browserstack.com/docs/app-automate/espresso/select-test-cases)
+      category: "Test Configuration"
+  - use_single_runner_invocation: false
+    opts:
+      title: "Single runner invocation"
+      summary: "Enable to run each test case in a single process to reduce overall build time."
+      description: "The single runner invocation feature runs each test case in a single process that reduces the overall build time. [Learn more.](https://www.browserstack.com/docs/app-automate/espresso/single-runner-invocation)"
+      value_options: 
+        - true
+        - false
+      category: "Test Configuration"
+  - use_mock_server: false
+    opts:
+      title: "Mock server"
+      summary: "Enable to mock a web server in your espresso tests to mock your API responses. Learn more."
+      description: "A mock web server mocks the behavior of an actual remote server. This makes it easy to test different scenarios without depending on your remote server and without having to make changes to your remote server. Local Testing, Network logs, and IP geolocation will not work if you are using a mock server. [Learn more.](https://www.browserstack.com/docs/app-automate/espresso/use-mock-server)"
+      value_options: 
+        - true
+        - false
+      category: "Test Configuration"
+  - check_build_status: true
+    opts:
+      title: "Build Status"
+      description: "Wait for build results"
+      summary: "Wait for BrowserStack to complete the execution and get the test results"
+      value_options: 
+        - true
+        - false
+      category: "Test Configuration"
+      # API params
+  - api_params:
+    opts:
+      title: "Test capabilities"
+      summary: New line key-value pairs of capabilities provided by BrowserStack.
+      description: |
+        New line key-value pairs of capabilities provided by BrowserStack.
+        
+        For example:
+        coverage=true
+        
+        geoLocation=CN
+        
+        [Learn about more capabilities.](https://www.browserstack.com/docs/app-automate/api-reference/espresso/builds#execute-a-build)
+      category: "Test Configuration"
+
```

Comment:
remove extra lines,

can we have a linter added in this repo as well? we can keep it as a dedicated task if not this one.

Reply:
removed extra lines and added a lint config